mkdocs.yml

```diff
@@ -21,11 +21,16 @@ nav:
       - Glossary: 'overview/glossary.md'
       - Logo assets: 'overview/logos.md'
   - Getting Started:
-      - Getting Started: 'getting-started/index.md'
-      - Create an App: 'getting-started/create-an-app.md'
-      - App configuration:
+      - Installing a standalone server (Developer or Admin): 'getting-started/standalone-install.md'
+      - Configuring Backstage (Admin):
+          - Database: 'getting-started/config/database.md'
+          - Authentication: 'getting-started/config/authentication.md'
           - Configuring App with plugins: 'getting-started/configure-app-with-plugins.md'
           - Customize the look-and-feel of your App: 'getting-started/app-custom-theme.md'
+          - Customizing your Homepage: 'getting-started/homepage.md'
+      - Logging in: 'getting-started/logging-in.md'
+      - Register a component: 'getting-started/register-a-component.md'
+      - Create a component: 'getting-started/create-a-component.md'
```

Comment:
Should these be in a `End User` section or something?

Reply:
A lof of these documentation pieces for end users are probably not going to be as valuable as we don't control what installations of backstage look like. A sidebar item might not be the same for every company, so it gets tricky to document this stuff in the first place.

Reply:
Yeah, my thinking here was to provide a "here's what the out of the box Backstage looks like/works like" section. Ideally, you should be referring back to the getting started documentation after you're already up and running. We should have separate documentation for those users.